state/lease/schema.go

```diff
@@ -172,13 +172,8 @@ func (doc clockDoc) skews(beginning, end time.Time) (map[string]Skew, error) {
 	// If it isn't, it could be ntp rolling the clock back slowly, so we add
```

Comment:
you didn't delete the check. it's not harmful, except it's second-guessing the definition of monotonicity